packages/kbn-check-mappings-update-cli/current_mappings.json

```
@@ -3078,6 +3078,19 @@
       }
     }
   },
+  "apm-custom-dashboards": {
+    "properties": {
+      "dashboardSavedObjectId": {
+        "type": "keyword"
+      },
+      "kuery": {
+        "type": "text"
+      },
+      "useServiceFilters": {
```

To avoid the React hook vibes:

```suggestion
      "serviceFiltersEnabled": {
```

Btw. should we split it up into a `serviceNameFilterEnabled` and `serviceEnvironmentFilterEnabled` in case someone needs to filter by `service.name` but not environment?